### ISSUE TITLE
pull metadata from PDF documents

### ISSUE THREAD

**Issue by raprasad** [IQSS/dataverse#3302]
Requires: filemetadata storage in place

---

Extract basic PDF metadata from PDF files
- number of pages, some text, etc
- existence of images/charts, etc
  There are over 67,796 PDF files in Dataverse.  
- Note: 1,227 are files with PDF extensions but not recognized as PDF.  (bad files?)
- We have a PDF preview (unreadable -- more like a thumbnail)

**Comment by raprasad** [comment]
dev note:

https://pdfbox.apache.org/

```
pip install pdfminer==20140328
```

**Comment by pdurbin** [comment]
Related to #2272.

**Comment by pdurbin** [comment]
Closing in favor of #2272
